browser/src/Plugins/Api/LanguageClient/LanguageClientHelpers.ts

```diff
@@ -25,9 +25,9 @@ export const ProtocolConstants = {
     },
 }
 
-export const wrapPathInFileUri = (path: string) => "file:///" + path
```

Comment:
Should try reverting this change, I don't believe it is necessary and may break Windows